src/app/pages/Home/Home.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styles from './Home.module.css';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import Typography from '../../components/Typography/Typography';
+import Navigation from '../../components/Navigation/Navigation';
+import ExplorationCardGroup from '../../components/ExplorationCardGroup/ExplorationCardGroup';
+import useLocalStorageSkills from '../../hooks/useLocalStorageSkills';
+import { Link, useParams } from 'react-router-dom';
+import { useHistory } from 'react-router';
+
+export default function Home(): JSX.Element {
+  const history = useHistory();
+  const params = useParams<{ title: string }>();
+
+  const onAddClick = () => console.log('added');
+
+  const explorationList = [
+    {
+      title: 'Crafting',
+      description: 'Exampledescription',
+      onClick: onAddClick,
+    },
+    {
+      title: 'Knitting',
+      description: 'Exampledescription',
+      onClick: onAddClick,
+    },
+    {
+      title: 'Art',
+      description: 'Exampledescription',
+      onClick: onAddClick,
+    },
+  ];
+  const { skills } = useLocalStorageSkills();
```

Comment:
It seems like you want to stick with this name, I'd add some coolness by adding the word "persistent". `usePersistentSkills` for example. Up to you